third_party/proton/tutorials/vector-add.py

```diff
@@ -0,0 +1,55 @@
+import torch
```

Comment:
Why do we need two files at all?

If we leave `pl.scope` but do not start profiling, they are considered as no-ops.

I think a `vector-add.py` file is enough.

We can use `proton --backend=instrumentation vector-add.py` without the `proton.start` function call

Reply:
I suppose we would probably want multiple tutorials then right? I think most users work flow is just call Proton directly from the python interface without using the proton script/binary. 

```python vector-add.py```

What do you think?

Reply:
But if we don't have proton.start then the IR is not modified when the DSL level is instrumented. I guess what I envision the majority of the workflow to look like is folks will instrument at the DSL level just to get a starting point, then move things around at the TTGIR level to get more fine grained control over the placement, and then use that to over ride the IR and run. So that's kind of what I'm trying to get at with this tutorial file.